### PR TITLE
Blender Cheat sheet: Corrected spelling mistakes.

### PR DESCRIPTION
**Pull request by mayank7**
Corrected spelling mistakes in blender cheat sheet.

IA Page: https://duck.co/ia/view/blender_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Blender Cheat Sheet](https://duck.co/ia/view/blender_cheat_sheet)

**Description**: Provide a cheatsheet for common Blender shortcuts

**Example Query**: [blender cheat sheet](https://beta.duckduckgo.com/?q=blender cheat sheet)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/blender_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/blender_cheat_sheet)*

File: share/goodie/cheat_sheets/json/blender.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "id": "blender_cheat_sheet",
     "name": "Blender",
     "description": "3D computer graphics software",
+    "aliases": ["blender"],
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/blender.json]: Remove this.

Comment (mayank7) [inline share/goodie/cheat_sheets/json/blender.json]: @GuiltyDolphin I removed it. 
Actually I did it to trigger CI build as build was failing for some reason for everyone. So after you people fixed it, I made a commit to trigger the build and check if it passes the tests.

**Comment by GuiltyDolphin** [comment]
Looks good to me @mayank7! :dolphin:

**Pull request merged**